tasks/db_seed.ts

```diff
@@ -40,17 +40,19 @@ const items = stories.map(({ by: userLogin, title, url, score, time }) => ({
   createdAt: new Date(time * 1000),
 })).filter(({ url }) => url);
 
+const users = Array.from(new Set(items.map((user) => user.userLogin)));
```

Comment:
```suggestion
const users = new Set(items.map((user) => user.userLogin));
```
`Set` is iterable. `Array.from()` is not needed 🙂

Reply:
> `Set` is iterable. `Array.from()` is not needed 🙂

Very true! Got confused because of mapping and filtering earlier on. Fixed.